project/src/Level 0/Spells.js

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from 'react'
+import Header from '../Level 1/Header'
+import Content from '../Level 1/Content'
+import Footer from '../Level 1/Footer'
+import SpellsList from '../Level 2/SpellsList'
+import querystring from 'query-string'
+import './Spells.css'
+
+
+export default class Spells extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      spells: [],
+      searchSpells: '',
+    }
+  }
+
+  componentDidMount() {
+    const url = 'https://www.potterapi.com/v1/spells';
+    const params = {
+        key: '$2a$10$UAdBgMAn4QwVG1Lmesy/IOcgM83yWR.5k9Q7rfUvSx/7vn58LmblK',
+    }
+    const qs = querystring.stringify(params);
+    fetch(`${url}?${qs}`)
```

Comment:
Could we have a function to handle the api calls? Makes sense to extract that logic?